conduct.md

```diff
@@ -9,11 +9,23 @@ testing fork
 
 ## Categories of Conduct
 
+* Conduct
+
+##Conduct (Alex Heinle and Megan Corletti)
+In this context, we will be defining conduct as the way a team member behaves
+and the way they act when carrying out their part of the project. First, everyone
+is obligated to adhere to the 9 categories in the Code of Conduct. They will be
+our foundation throughout the semester when working together. Moving on, everyone
+should respect one another; no one should be excluded based off of race or gender.
+Team members should not shun the ideas of anyone, especially those who may
+have not have had an equal amount of experience as others. Also, at the beginning
+of each session, everyone should enter with positive attitudes, These positive
+attitudes should stay consistent whether the project is working or breaking.
+
 * Attendance
 * Deadlines
 * Communication
 * Participation
-* Conduct
```

Comment:
We should try to keep the order of the sections consistent across branches so merging is simple when each PR is merged.